src/routes/phenomenon.route.js

```diff
@@ -0,0 +1,6 @@
+const express = require("express");
+const app  = express();
+
+app.get("http://localhost:3000/phenomenon/" , (res, req) => {
+    app.post()
```

Comment:
You can't post from the back end, you must modify the response object to have a status and either send a json message or some text.

Reply:
Also, remember to not hard code addresses like this, it should be "/phenomenon" with no trailing "/"